DataFormats/PatCandidates/interface/MET.h

```diff
@@ -200,11 +247,25 @@ namespace pat {
       // ---- holder for pfMET specific info ---
       std::vector<SpecificPFMETData> pfMET_;
 
+      // MET significance
+      double metSig_;
+      
+      const PackedMETUncertainty findMETTotalShift(MET::METCorrectionLevel cor, MET::METUncertainty shift) const;
+   
+      std::map<MET::METCorrectionLevel, std::vector<MET::METCorrectionType> > corMap_;
```

Comment:
Uhm, so, but then I think I need to read the code more accurately to
understand what is really done and stored.

On Tue, May 19, 2015 at 4:14 PM, mmarionncern <notifications@github.com>
wrote:

> In DataFormats/PatCandidates/interface/MET.h
> <https://github.com/cms-sw/cmssw/pull/9117#discussion_r30602806>:
>
> > @@ -200,11 +247,25 @@ namespace pat {
> >        // ---- holder for pfMET specific info ---
> >        std::vector<SpecificPFMETData> pfMET_;
> >
> > +      // MET significance
> > +      double metSig_;
> > +
> > +      const PackedMETUncertainty findMETTotalShift(MET::METCorrectionLevel cor, MET::METUncertainty shift) const;
> > +
> > +      std::map<MET::METCorrectionLevel, std::vector<MET::METCorrectionType> > corMap_;
>
> This is a transient map, not stored in files, only used when the
> combination of corrections is done. Should I still proceed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/9117/files#r30602806>.
>